src/sentry/static/sentry/app/components/events/interfaces/breadcrumbsV2/breadcrumbDescription/breadcrumbDescription.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import BreadcrumbDescriptionDefault from './breadcrumbDescriptionDefault';
+import BreadcrumbDescriptionException from './breadcrumbDescriptionException';
+import BreadcrumbDescriptionHttp from './breadcrumbDescriptionHttp';
+import {Breadcrumb, BreadcrumbType} from '../../breadcrumbs/types';
```

Comment:
👎 on negative directory traversal in imports. But could just be me.

Reply:
what? I didn't get it